README.md

```diff
@@ -6,32 +6,19 @@
 [![Maven Central](https://img.shields.io/maven-central/v/com.auth0.android/auth0.svg?style=flat-square)](https://search.maven.org/artifact/com.auth0.android/auth0)
 [![javadoc](https://javadoc.io/badge2/com.auth0.android/auth0/javadoc.svg)](https://javadoc.io/doc/com.auth0.android/auth0)
 
-Easily integrate Auth0 into Android apps. Add **login** and **logout**, store **credentials** securely, and access **user information**.
-
-## Table of contents
-
-- [Requirements](#requirements)
-- [Installation](#installation)
-   + [Permissions](#permissions)
-- [Getting Started](#getting-started)
-   * [Authentication with Universal Login](#authentication-with-universal-login)
-   * [Clearing the session](#clearing-the-session)
-- [Next steps](#next-steps)
-   * [Authentication API](#authentication-api)
-   * [Management API](#management-api)
-   * [Token Validation](#token-validation)
-   * [Organizations](#organizations)
-- [Credentials Manager](#credentials-manager)
-   * [Basic](#basic)
-   * [Encryption enforced](#encryption-enforced)
-- [Networking client customization](#networking-client-customization)
-- [FAQ](#faq)
-- [Proguard](#proguard)
-- [What is Auth0?](#what-is-auth0)
-- [Issue Reporting](#issue-reporting)
-- [License](#license)
-
-## Requirements
+📚 [Documentation](#documentation) • 🚀 [Getting Started](#getting-started) • ⏭️ [Next Steps](#next-steps) • 💬 [Feedback](#feedback)
```

Comment:
Might be best to use the the emoji shortcuts (e.g., `:books:`) instead of pasting the emoji directly in, for MD parsers.